### ISSUE TITLE
PCB spacing on blend Micro

### ISSUE THREAD

**Issue by brookpatten** [makenai/FritzingParts#1]
Just FYI, the pin spacing for the blend micro in the PCB layout is not correct.  It seems like it is spaced .08 mil rather than the standard .1 mil.  I'm not an expert on this so I could be wrong, but if you layout a board with something like a header you'll see the alignment issue.

If you print a board using this part I don't think it's going to line up.

**Comment by Ehryk** [comment]
Yep, I just spent $45 on boards with the incorrect pin spacing on the Blend Micro, and also the bottom layer isn't connected.
